### DISCUSSION TITLE
Does turbo cache replace the need for other tool caching?

### DISCUSSION THREAD

**Discussion by tomdavidson** [vercel/turborepo#2021] [Help] [answered]
Hello, does turbo's caching completely replace the need tools invoked by turbo to cache, such as jest, eslint, and prettier?

**Comment by anthonyshew** [comment, marked as answer]
Turborepo caches at the workspace level. So, if you miss a cache according to Turborepo, your task will run in its entirety.

Therefore, if your task does run, you could choose to use the cache from your various tools to reduce their run times (when they do run).
